utilities/data/getStoryblokRedirects.mjs

```diff
@@ -49,7 +49,7 @@ const sanitizeRedirectCode = (code) => {
  * @returns {string}
  */
 const sanitizeSourcePath = (str) => {
-  const parts = str.split('?');
+  const parts = str?.split('?');
```

Comment:
If `str` isn't a string, it shouldn't have been passed in. I feel that the optional chaining operator should not be here.

Reply:
Yeah that was while I was troubleshooting while I first got the built error because of the empty redirects. The superficial error points to the split so this was my first attempt. I think we don't need these now - I'll remove.